lib/mongoose_push.ex

```diff
@@ -96,16 +105,21 @@ defmodule MongoosePush do
 
   defp maybe_log({:error, {type, reason}} = return_value) do
     Logger.warn(
-      ~s"Unable to complete push request due to service error: #{inspect(reason)} in category: #{
-        type
-      }"
+      fn ->
+        "push"
+      end,
+      result: :error,
+      log: :command,
+      reason: reason,
+      type: type
     )
 
     return_value
   end
 
   defp maybe_log({:error, reason} = return_value) do
-    Logger.warn(~s"Unable to complete push request due to #{inspect(reason)}")
+    Logger.warn(fn -> "push" end, result: :error, log: :command, reason: reason)
```

Comment:
For message I would suggest - "Unable to complete push request". In general messages should be "human readable", but should not contain any variables. It's actually very similar to events name in telemetry, only stringified.